src/commands/BaseCommand.ts

```diff
@@ -17,4 +17,10 @@ export abstract class BaseCommand {
   protected get exitCode(): number {
     return process.exitCode || 0;
   }
+
+  protected handleError(error: Error, exitCode: number) {
+    this.writeError(error.message);
+    this.exitCode = 1;
+    return Promise.reject(error);
+  }
```

Comment:
This makes the return type `Promise<never>` 👍 .

![image](https://user-images.githubusercontent.com/1885333/43106969-4efbd448-8eb1-11e8-8e7b-9e9e578f2052.png)

Reply:
I tried this before although I put `Promise<void>` as the return type and it corrupted the `beautify` method.